website/docs/guides/requests/2-insert-ttl-data.js

```diff
@@ -0,0 +1,11 @@
+db.runCommand({
+  insert: 'books',
+  documents: [
+    {
+      title: 'The Great Gatsby',
+      publication: {
+        date: new Date()
```

Comment:
JS files will be generated from static JSON files by the CTS tool, and that line cannot be generated.

(not to mention that this example is just weird)